std/process.d

```diff
@@ -555,6 +560,7 @@ private Pid spawnProcessImpl(scope const(char[])[] args,
                         close(i);
                     }
                 }
+                free(pfds);
```

Comment:
Why? The current process is either getting replaced by `execve`, or killed by exiting with an error.

Reply:
Ah yes, remove this.